app/propsGenerators/indexPagePropsGenerator.ts

```diff
@@ -11,21 +11,31 @@ import {
   IndexPageProps,
 } from "../types/IndexPageProps";
 
+function calculateDosePercentage(
+  dosesCount: number,
+  totalPopulation: number
+): number {
+  return parseFloat(((dosesCount / totalPopulation) * 100).toFixed(6));
```

Comment:
`toFixed` still rounds, but 6dp should be "enough" precision for us to not have it cause an issue